src/lib/xiaomi.ts

```diff
@@ -1348,13 +1348,41 @@ export const trv = {
 export const manufacturerCode = 0x115f;
 
 export const xiaomiExtend = {
-    switchType: extend.enumLookup({
+    switchType: (args?: KeyValueAny) => extend.enumLookup({
         name: 'switch_type',
         lookup: {'toggle': 1, 'momentary': 2, 'none': 3},
         cluster: 'aqaraOpple',
         attribute: {id: 0x000a, type: 0x20},
         description: 'External switch type',
         zigbeeCommandOptions: {manufacturerCode},
+        ...args,
+    }),
+    powerOnBehavior: (args?: KeyValueAny) => extend.enumLookup({
+        name: 'power_on_behavior',
+        lookup: {'on': 0, 'previous': 1, 'off': 2},
+        cluster: 'aqaraOpple',
+        attribute: {id: 0x0517, type: 0x20},
+        description: 'Controls the behavior when the device is powered on after power loss',
+        zigbeeCommandOptions: {manufacturerCode},
+        ...args,
+    }),
+    operationMode: (args?: KeyValueAny) => extend.enumLookup({
+        name: 'operation_mode',
+        lookup: {'decoupled': 0, 'control_relay': 1},
+        cluster: 'aqaraOpple',
+        attribute: {id: 0x0200, type: 0x20},
+        description: 'Decoupled mode for relay',
+        zigbeeCommandOptions: {manufacturerCode},
+        ...args,
+    }),
+    action: (args?: KeyValueAny) => extend.enumLookup({
```

Comment:
```suggestion
    action: (args?: ActionEnumLookupArgs) => extend.enumLookup({
```

I've added typing, please use that instead of `KeyValueAny`